src/Evolution/DgSubcell/Tags/InitialTciData.hpp

```diff
@@ -44,5 +47,24 @@ struct InitialTciData {
                << direction_and_element_id.second << " more than once");
     current_inbox.emplace(std::forward<ReceiveDataType>(data));
   }
+
+  static std::string output_inbox(const type& inbox,
+                                  const size_t padding_size) {
+    std::stringstream ss{};
+    const std::string pad(padding_size, ' ');
+
+    ss << std::scientific << std::setprecision(16);
+    ss << pad << "InitialTciDataInbox:\n";
+    for (const auto& [current_time_step_id, hash_map] : inbox) {
+      ss << pad << " Current time: " << current_time_step_id.substep_time()
+         << "\n";
```

Comment:
You probably want the whole id, not just the substep time, since the latter aren't uniquely identifying.

Same comment everywhere else.

Reply:
Is there a `double` that *does* uniquely identify what time we are at? I ask because the printed version of a `TimeStepId` is quite difficult to interpret/understand at a glance, but a `double` is simple so I think it'd be easier for people (me) if I just compared `double`s. Also, we may need to compare these `TimeStepId`s to the FoT expirations (and potentially other times represented as `double`s) and that's difficult to do with the normal stream of a `TimeStepId`.

Reply:
There isn't, but the substep time is printed at the end of the TimeStepId output:
```
5:Slab[1.34257480385833006,4.67348735890730005]:1/4:1:2.59166701200169403
                                                      ^^^^^^^^^^^^^^^^^^^
```
Still a bunch of noise you have to ignore, but you don't have to do any calculations or anything.